engine/table/src/main/java/io/deephaven/engine/table/impl/lang/QueryLanguageFunctionUtils.java

```diff
@@ -15,6 +15,8 @@
 
 import static java.lang.Math.*;
 
+import java.lang.Number;
```

Comment:
This should not be necessary, all java.lang imports are implicit, unless there is a conflicting Number type?